openff/toolkit/topology/topology.py

```diff
@@ -31,7 +31,6 @@
 
 import numpy as np
 from numpy.typing import NDArray
-from openff.units import ensure_quantity
```

Comment:
This method is in `openff.units.openmm`, which means importing it results in an `import openmm` call whenever anything in this module is imported. Obviously it's going to be in memory for many use cases, but it's unnecessary to import it here